static/js/containers/App.js

```diff
@@ -88,49 +85,19 @@ class App extends React.Component {
       location: { pathname },
       dispatch,
     } = this.props;
-    let [ complete, info ] = validateProfileComplete(profile);
+    let [ complete, url, errors] = validateProfileComplete(profile);
     if (
       userProfile.getStatus === FETCH_SUCCESS &&
       profile.agreed_to_terms_of_service &&
       !PROFILE_REGEX.test(pathname) &&
       !complete
     ) {
-      const { url, title, text } = info;
-      dispatch(updateDialogText(text));
-      dispatch(updateDialogTitle(title));
-      dispatch(setDialogVisibility(true));
+      dispatch(startProfileEdit);
```

Comment:
Found the problem - syntax error. `startProfileEdit` is a function, so we need to change this line to:

```js
dispatch(startProfileEdit())
```

then it should work fine.